jobboard/urls.py

```diff
@@ -0,0 +1,11 @@
+__author__ = 'ernesto'
+from .views import JobList, JobDetail
+from django.conf.urls import patterns, url
+from django.utils.translation import ugettext_lazy as _
+
+
+urlpatterns = patterns(
+    'jobboard.views',
+    url('^board/list/$', view=JobList.as_view(), name='job_list'),
```

Comment:
why not just matching the empty string to append to `jobs` ?

Reply:
done